waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/service/impl/PrefixBasedDatabaseMappingServiceTest.java

```diff
@@ -42,6 +44,7 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
+import org.tuckey.web.filters.urlrewrite.utils.StringUtils;
```

Comment:
Is this really the `StringUtils` you want to use?

Reply:
lol